src/main/java/uk/ac/cam/cl/quebec/face/MessageProcessor.java

```diff
@@ -24,6 +29,7 @@
     static{ System.loadLibrary(Core.NATIVE_LIBRARY_NAME); }
 
     private static final String localTrainingFile = "/tmp/opencv-training.yaml";
+    private static final double recognitionThreshold = 1000;
```

Comment:
How much nearer to the beginning of the file do you want it? It's the 3rd thing in the class